copier/cli.py

```diff
@@ -168,6 +171,20 @@ def data_switch(self, values: StrSeq) -> None:
             key, value = arg.split("=", 1)
             self.data[key] = value
 
+    @cli.switch(
+        ["--data-file"],
+        cli.ExistingFile,
+        help="Load data from a YAML file",
+        excludes=["--data"],
+    )
+    def data_file_switch(self, path: LocalPath) -> None:
+        """Update [data][] with provided values.
+        Arguments:
```

Comment:
Could you please add an empty line here?

```suggestion

        Arguments:
```